crates/forge_domain/src/chat_response.rs

```diff
@@ -124,6 +124,8 @@ pub struct TitleFormat {
     pub title: String,
     pub sub_title: Option<String>,
     pub category: Category,
+    pub usage: Option<Usage>,
+    pub token_limit: Option<usize>,
```

Comment:
TitleFormat doesn't need to change. WE should implement a format function for printing the logs. The logs should collect all the relevant data from different places for eg: message from title, usage from conversation etc. and then format it as per the env variable, and print it.

Reply:
The env variable used to format the log message should be used in zsh plugin as well as in REPL.